checkstyle.xml

```diff
@@ -54,13 +56,15 @@
             <property name="tokens" value="LITERAL_TRY, LITERAL_FINALLY, LITERAL_IF, LITERAL_ELSE, LITERAL_SWITCH"/>
         </module>
         <module name="NeedBraces"/>
-        <module name="LeftCurly">
-            <property name="maxLineLength" value="100"/>
```

Comment:
Why did you remove the limit? What's the default?

Reply:
I've only replaced the file with current config we have in recent project aka send-letter-service

Reply:
it has been deprecated (guessing since v8) http://checkstyle.sourceforge.net/config_blocks.html#LeftCurly